web-api/src/cases/getCasesForUserLambda.js

```diff
@@ -6,9 +6,9 @@ const { genericHandler } = require('../genericHandler');
  * @param {object} event the AWS event object
```

Comment:
Another small nit in a similar vein.  Line 4 mentions this is used to fetch all open cases, but it also returns closed cases too.  I can't comment directly on line 4 since it's outside of what GitHub lets me.